context/app/config.py

```diff
@@ -1,5 +1,6 @@
 types = {
     'donor': {'display_name': 'Donors', 'in_header': True},
     'sample': {'display_name': 'Samples', 'in_header': True},
-    'dataset': {'display_name': 'Datasets', 'in_header': True}
+    'dataset': {'display_name': 'Datasets', 'in_header': True},
+    'collection': {'display_name': 'Collections', 'in_header': True}
```

Comment:
This file is vestigial. Filed #1374 to clean up.

Reply:
Thanks!